vector/src/main/java/im/vector/app/features/login2/LoginWebFragment2.kt

```diff
@@ -124,8 +124,10 @@ class LoginWebFragment2 @Inject constructor(
         views.loginWebWebView.loadUrl(url)
 
         views.loginWebWebView.webViewClient = object : WebViewClient() {
-            override fun onReceivedSslError(view: WebView, handler: SslErrorHandler,
-                                            error: SslError) {
+            override fun onReceivedSslError(
+                    view: WebView, handler: SslErrorHandler,
```

Comment:
One line per parameter ? :)

Reply:
I've not found a way to do this either via editorconfig. If they were already previously formatted like this, they'd stay this way even after autoformatting.

I separated the method parameters you brought up in your comments but I can't do this app-wide